### PR TITLE
Homepage updates: question finder, resources, stories and benefits sections

### PR DESCRIPTION
**Pull request by mwtrew**
- Styling and copy changes to resources, stories and benefits sections
- Simplify benefits section
- Add question finder slice, which is also added to Topics page

### CODE REVIEW

**Review by jacbn — COMMENTED** [review]
Added some minor suggestions for style and have committed two other fixes:
- minified the two SVGs a with b64-encoded image inside
- replaced Show Topics buttons on the cards with just one button underneath, as they all led to the same place.

@mlt47 happy to merge if the question finder slice is all good.

File: src/app/components/pages/AllTopics.tsx [hunk 1]
```diff
@@ -85,5 +85,31 @@ export const AllTopics = () => {
...
             </Row>
         </Container>
+        <section id={'topics-question-finder'}>
+            <Container className={"mb-5 p-5 mx-auto"}>
+                <Row className={"align-items-center justify-content-center"}>
+                    <Col xs={12} lg={6}>
+                        <h2 className={"font-size-1-75 mb-4"}>Check your understanding</h2>
+                        <p>
+                            Using the Question finder you can select topics and concepts and we’ll generate a random
+                            selection questions for you to use. You can also specify your level and exam board.
```
Comment (mlt47) [inline src/app/components/pages/AllTopics.tsx:95]:
```suggestion
                            selection of questions for you to use. You can also specify your level and exam board.
```

File: src/app/components/pages/AllTopics.tsx [hunk 2]
```diff
@@ -85,5 +85,31 @@ export const AllTopics = () => {
                 </Col>
             </Row>
         </Container>
+        <section id={'topics-question-finder'}>
+            <Container className={"mb-5 p-5 mx-auto"}>
+                <Row className={"align-items-center justify-content-center"}>
+                    <Col xs={12} lg={6}>
+                        <h2 className={"font-size-1-75 mb-4"}>Check your understanding</h2>
+                        <p>
+                            Using the Question finder you can select topics and concepts and we’ll generate a random
```
Comment (mlt47) [inline src/app/components/pages/AllTopics.tsx:94]:
```suggestion
                            Using the question finder you can select topics and concepts and we’ll generate a random
```

File: src/app/components/pages/AllTopics.tsx [hunk 3]
```diff
@@ -85,5 +85,31 @@ export const AllTopics = () => {
...
+                            selection questions for you to use. You can also specify your level and exam board.
+                        </p>
+                        <p><b>Students</b> might want to try our Question Finder to revise and get feedback on any
```
Comment (mlt47) [inline src/app/components/pages/AllTopics.tsx:97]:
```suggestion
                        <p><b>Students</b> might want to try our question finder to revise and get feedback on any
```

File: src/app/components/pages/AllTopics.tsx [hunk 4]
```diff
@@ -85,5 +85,31 @@ export const AllTopics = () => {
...
+                            mistakes.</p>
+                        <p><b>Teachers</b> might want to try our Question Finder to create quizzes and assign them to
```
Comment (mlt47) [inline src/app/components/pages/AllTopics.tsx:99]:
```suggestion
                        <p><b>Teachers</b> might want to try our question finder to create quizzes and assign them to
```

**Review by mlt47 — COMMENTED** [review]
Suggesting some minor copy edit changes. Will check for original copy during today's tech meeting.

**Pull request merged**